crates/js-sys/src/lib.rs

```diff
@@ -3561,6 +3561,33 @@ pub mod WebAssembly {
         pub fn set(this: &Table, index: u32, function: &Function) -> Result<(), JsValue>;
     }
 
+    
+    // WebAssembly.Global
+    #[wasm_bindgen]
+    extern "C" {
+        /// The `WebAssembly.Global()` constructor creates a new `Global` object
+        /// of the given type and value.
+        ///
+        /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Global)
+        #[wasm_bindgen(js_namespace = WebAssembly, extends = Object, typescript_type = "WebAssembly.Global")]
+        #[derive(Clone, Debug, PartialEq, Eq)]
+        pub type Global;
+
+        /// The `WebAssembly.Global()` constructor creates a new `Global` object
+        /// of the given type and value.
+        ///
+        /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Global)
+        #[wasm_bindgen(constructor, js_namespace = WebAssembly, catch)]
+        pub fn new(global_descriptor: &Object, value: Option<usize>) -> Result<Global, JsValue>;
+
+        /// The value prototype property of the `WebAssembly.Global` object
+        /// returns the value of the global.
+        ///
+        /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Global)
+        #[wasm_bindgen(method, getter, js_namespace = WebAssembly)]
+        pub fn value(this: &Global) -> JsValue;
```

Comment:
Should this return `f64` ?

Reply:
I think `JsValue` here is fine for `externref`, but can this also add a setter?

Reply:
Done. Added a setter as well.
Note that I also added the `structural` attr to value, but it would be great to assure that's the correct way